utils/hct/hctdb.py

```diff
@@ -8016,8 +8016,9 @@ def build_valrules(self):
         )
         self.add_valrule("Meta.ValidSamplerMode", "Invalid sampler mode on sampler .")
         self.add_valrule(
```

Comment:
Oops, needed other function for separate message.
```suggestion
        self.add_valrule_msg(
```